### ISSUE TITLE
Cannot see Team Fortress 2 in my library.

### ISSUE THREAD

**Issue by rxchiie** [FriendsOfGalaxy/galaxy-integration-steam#116]
**Describe the bug**
After installing TF2, and linking my Steam account, it didn't show in GOG Galaxy.

**Expected behavior**
I expected TF2 to appear under the installed category.

**Attached logs**
[logs.zip](https://github.com/FriendsOfGalaxy/galaxy-integration-steam/files/6287362/logs.zip)
Here you go, the whole pack of logs for you.

**Notes**
I noticed that when I added the game manually, it would appear as Other first, but then it would change to Steam and run just fine, like it synced like the other Steam games, HOWEVER, restarting GOG Galaxy makes it delete itself or go back to Other.

**Comment by K1LL3RPUNCH** [comment]
+1

**Comment by K1LL3RPUNCH** [comment]
https://github.com/FriendsOfGalaxy/galaxy-integration-steam/issues/76

**Comment by rxchiie** [comment]
Seems the issue is resolved. This morning opened up Galaxy and TF2 synced like a minute after opening. HOWEVER, I am not sure if this issue should be closed, since the issue could reoccur.

**Comment by rxchiie** [comment]
Just tried re-opening Galaxy, I saw TF2 appearing for a second during syncing, and then disappearing. When I tried to sync again manually, it didn't appear anymore. Not even a minute after opening Galaxy. This issue is really weird.

**Comment by rxchiie** [comment]
Ah-ha! Ran TF2 from the start menu and it appeared in Galaxy! It stayed there even after closing the game, restarting Galaxy, however, occured in TF2 not showing anymore in the library.

**Comment by FriendsOfGalaxy** [comment]
Hey @rxchiie,

My suspicion is the problem with games cache in the plugin.
- Steam send us games for a long time and we have to timeout it as certain point and send already fetched games to Galaxy
- later on we send the rest of them when they come
- every time we update licences cache

There is probably a bug with cache or our approach is wrong. We should cache already resolved games and return them immediately on Galaxy start. Cache update should be done independently. There may be some problems with handling removed games though.

I have a strong feeling that the issue is related to #38

**Comment by rxchiie** [comment]
@FriendsOfGalaxy So, I understand there's nothing the end-users can do, and it's just that you have to fix the bug?

**Comment by FriendsOfGalaxy** [comment]
@rxchiie Yes, almost nothing. Raising attention to this is already a thing, thanks for that. 

Any 👍 for the issue #80 will be also helpful to catch most common/painful bugs.

And sometimes testing a fix build - I'll ask for help in case I do not have clear bug reproduction.
